public/app/features/alerting/unified/components/settings/AlertmanagerConfig.test.tsx

```diff
@@ -75,6 +77,7 @@ describe('vanilla Alertmanager', () => {
 
   beforeEach(() => {
     setupVanillaAlertmanagerServer(server);
+    setupDataSources(...Object.values(mockDataSources));
```

Comment:
I removed the datasource logic from `setupVanillaAlertmanagerServer`, so had to add it into here instead (`setupVanillaAlertmanagerServer` will be removed eventually)